src/sentry/static/sentry/app/views/releasesV2/detail/releaseStat.tsx

```diff
@@ -21,15 +26,23 @@ const Wrapper = styled('div')`
   }
 `;
 
-const Label = styled('div')`
+const Label = styled('div')<{hasHelp: boolean}>`
   font-weight: 600;
   font-size: ${p => p.theme.fontSizeSmall};
   text-transform: uppercase;
   color: ${p => p.theme.gray500};
   line-height: 1.3;
   margin-bottom: ${space(0.25)};
   white-space: nowrap;
+  padding-right: ${p =>
+    p.hasHelp ? `calc(${space(0.25)} + ${p.theme.iconSizes.xs})` : '0'};
 `;
+
+const StyledQuestionTooltip = styled(QuestionTooltip)`
+  margin-left: ${space(0.5)};
+  position: absolute;
```

Comment:
Why do the icons need to be absolute positioned? In other places we've used a margin on the left side of the icon to add spacing and used flexbox to do alignment.

Reply:
They don't, that was a brainfart on my end. Thank you.